### ISSUE TITLE
[Feature Request] Add show from tv calendar website

### ISSUE THREAD

**Issue by djack42** [SiCKRAGETV/sickrage-issues#1684]
I use the website http://www.pogdesign.co.uk/cat/ (tv calendar) to track new shows and i'm looking for a way to add a new show to sickrage based on the shows selected on my tvcalendar account if they are missing from sickrage.
This website will get a huge update in few weeks, adding a lot of new shows and it is for me a really good tool. So we have time to debate.
There is no api as far as i know, but one page list all shows with the selected status, so i don't think it is too hard to parse. There is also a page with all additions, really usefull.
This website only list active shows, ended ones are purged after a while.

Possible evolutions : watched episodes (i don't use it), trailer link, renewal and cancellation estimations based on ratings, statistics of popular shows and recently selected and selection growth.

**Comment by neoatomic** [comment]
@djack42 From a practical view, the site doesn't add anything over TheTVDB or TVRage.
So i'm a little skeptical adding this. Especially as it hasn't an API.

**Comment by WebSpider** [comment]
I'd have to agree: Why add something that we already have, using TVDB/TVRage and Trakt?

**Comment by OmgImAlexis** [comment]
We have switch to using feathub for feature requests. If this feature is still important to anyone, please
check that it has not been added and add it [HERE](http://feathub.com/SiCKRAGETV/SickRage)
Make sure to vote!
